src/zincr.ts

```diff
@@ -31,23 +31,33 @@ export class Zincr {
     const issue = context.issue();
     const { sha } = context.payload.pull_request.head;
 
+    const valid_pr_actions = ["opened", "edited", "reopened","synchronize"];
+
+
    
     // if there is no pull request or the state is not open, no reason to continue
     if (!pullRequest || pullRequest.state !== "open") return;
 
+    // only trigger on valid PR events
+    if (
+      event == "pull_request" &&
+      valid_pr_actions.indexOf(context.payload.action) < 0
```

Comment:
Doesn't this mean "if(Event is PR && the Action in the Payload is not a Valid PR Action)" ? Seems different then the comment of "only trigger on valid PR Events". Shouldn't this be ` valid_pr_actions.indexOf(context.payload.action) >= 0 `

Reply:
Ignore, I see you don't return stuff, so the code comment is just a bit confusing.